### DISCUSSION TITLE
Tea list

### DISCUSSION THREAD

**Discussion by DesmoricFR** [ncosgray/cuppa_mobile#64] [Q&A] [answered]
I would like to édit the tea list to add more tea by default. 

1. How can I do this to share withe the dev team (I’m not a dev, just want to help) ?
2. How to "format" the file (rows and lines, number of caracters by row, etc.) ?

Regards

**Comment by ncosgray** [comment]
Thank you for starting this conversation! I designed Cuppa to be as basic and simple as possible for the first-time user. That's why I chose just 3 generic initial teas. But I am certainly open to the idea of adding more default teas to Cuppa, if there are popular categories of tea that are completely missing.

What are your suggestions for additional defaults besides the current Black, Green, and Herbal tea options? Perhaps other Cuppa users who visit this discussion board can weigh in as well.

**Reply by DesmoricFR** [reply]
I made a generic list of teas with infusion times (ods joined file, in french but I can translate it in English if you need 
[teas.ods](https://github.com/ncosgray/cuppa_mobile/files/7944319/teas.ods))
If you consider it as too long, no problem, it's just a proposal :)

**Reply by ncosgray** [reply]
Many of these are teas I'm not familiar with! Do you have English translations too?

**Reply by DesmoricFR** [reply]
So, I translate the file for you to understand.
Regards
[Thes-tea_times.ods](https://github.com/ncosgray/cuppa_mobile/files/8027362/Thes-tea_times.ods)

**Reply by ncosgray** [reply]
This is wonderful! I plan to add preset tea options to a future version of Cuppa. Instead of just adding a new blank tea you'll have the option to choose from a list of predefined choices. Your list of teas will be the perfect starting point for this. Thank you!

**Comment by ncosgray** [comment, marked as answer]
In version 1.6 (to be released soon) I've added a list of preset tea options to the Add Tea dialog box. Many of the tea presets were taken from or inspired by the tea list you shared. Thank you so much!

![app-prefs-addtea](https://user-images.githubusercontent.com/13181064/172030165-5d321c4d-641d-43f7-9c71-29856822cea7.png)

**Reply by DesmoricFR** [reply]
I saw it with the update, nice work 👍

**Comment by teolemon** [comment]
@ncosgray You could add the possibility to scan tea barcodes with Open Food Facts and contribute missing tea bags to the database.
We have an official implementation at https://github.com/openfoodfacts/smooth-app as well as a Dart package: https://github.com/openfoodfacts/openfoodfacts-dart

**Reply by ncosgray** [reply]
This is amazing! Thank you for bringing the Open Food Facts project to my attention. I will take a closer look at this in the coming weeks.
